pkg/api/openapi/model_resource.go

```diff
@@ -31,7 +31,7 @@ type Resource struct {
 	DeletedAt      *time.Time             `json:"deleted_at,omitempty"`
 	Manifest       map[string]interface{} `json:"manifest,omitempty"`
 	DeleteOption   map[string]interface{} `json:"delete_option,omitempty"`
-	UpdateStrategy map[string]interface{} `json:"update_strategy,omitempty"`
+	ManifestConfig map[string]interface{} `json:"manifest_config,omitempty"`
```

Comment:
I thought we only also add GroupResource into the API.

Reply:
done.